server_admin/topics/sessions/transient.adoc

```diff
@@ -0,0 +1,11 @@
+[[_transient-session]]
+
+=== Transient sessions
+
+{project_name} has concept of transient sessions. When transient sessions are used, there is no real user session created after successful authentication.
+There is just temporary "transient" session created for the scope of the current request, which successfully authenticated user. This allows
+to run <<_protocol-mappers, protocol mappers>> after the authentication.
+
+When transient sessions are used, the client application does not have a way to refresh token, introspect token or check anyhow if particular session is valid.
```

Comment:
Replace "application does not have a way to refresh token, introspect token or check anyhow if  particular session..."  
with
"application has no way to refresh or introspect the token or check if a specific session..."